lib/atat-web-api-stack.ts

```diff
@@ -196,17 +209,17 @@ export class AtatWebApiStack extends cdk.Stack {
       );
     });
 
-    // Cloudwatch Logs for C5ISR
-    const logGroup = new logs.LogGroup(this, "LogGroup", {
-      logGroupName: `${environmentName.toLowerCase()}-cssp-cwl-logs`,
-      retention: RetentionDays.INFINITE,
-    });
-    NagSuppressions.addResourceSuppressions(logGroup, [
-      {
-        id: "NIST.800.53.R4-CloudWatchLogGroupRetentionPeriod",
-        reason: "Setting retention to infinte so no minimum retention is needed. ",
-      },
-    ]);
+    // Cloudwatch Log group for C5ISR
+    // const logGroup = new logs.LogGroup(this, "LogGroup", {
```

Comment:
yea so that basically created the log group for C5ISR, so it just needs to run once (it already has) and now I commented it out so that the following cdk deploy will work. CDK deployan only run this once or else itll fail and say a log group was  already created with that name. Commenting these lines out wont delete the log group. cdk basically is a one time deal when making log group in cloudwatch.

Reply:
OK so can we set this up in a more resilient way such that we can determine programmatically if the section is necessary versus manually commenting it out? Unless I am not understanding something here?

Reply:
Jeff, made the changes. just had to move the snippet to the shared data stack to not run into the resource creation issue.